src/styles/stopwatch.scss

```diff
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: $bodyBackground-color;
+}
+
+.watch {
+  display: flex;
+  width: $watch-size;
+  height: $watch-size;
+  box-shadow: 5px 7px 10px -1px rgba(166, 166, 166, 0.25) inset,
+    -6px -1px 10px 2px rgba(255, 255, 255, 0.5) inset,
+    -6px -7px 20px 0 rgba(255, 255, 255, 0.5),
+    8px 8px 20px 0 rgba(170, 170, 170, 0.25);
+  border: 1px solid rgba(63, 63, 63, 0.1);
+  border-radius: $radius;
+  justify-content: center;
+  align-items: center;
+
+  &__center {
+    z-index: 1;
+    width: $watchCenter-size;
+    height: $watchCenter-size;
+    background-color: #272727;
+    border-radius: $radius;
+  }
+
+  &__second-hand {
+    position: absolute;
```

Comment:
looks good but I don't see `position: relative;` on parent element. where do you position that from?

Reply:
> looks good but I don't see `position: relative;` on parent element. where do you position that from?

It is not clear why, but it positioned elements relative to the parent element without this setting